Source/Core/Core/IOS/USB/Emulated/Skylander.h

```diff
@@ -19,7 +19,36 @@ constexpr u8 MAX_SKYLANDERS = 16;
 
 namespace IOS::HLE::USB
 {
-extern const std::map<const std::pair<const u16, const u16>, const char*> list_skylanders;
+enum Game
+{
+  SPYROS_ADV,
+  GIANTS,
+  SWAP_FORCE,
+  TRAP_TEAM,
+  SUPERCHARGERS,
+  G_OTHER,
+};
+enum Element
+{
+  MAGIC,
+  FIRE,
+  AIR,
+  LIFE,
+  UNDEAD,
+  EARTH,
+  WATER,
+  TECH,
+  E_OTHER,
+};
```

Comment:
I realize that's probably quite annoying to change, but... can you make those into `enum class`es instead of plain `enum`s?